### PR TITLE
[For 2.6.3] fix(back): add 2 more index in content_revisions database table

### PR DESCRIPTION
**Pull request by inkhey**
related to #2787 

this fix add 2 new index in content_revisions. This allow database to retrieve more effectively
content_revisions needed.

New index are on existing foreign keys:
- content_id
- workspace_id

## Checkpoints

The goal of this section is to help code reviewer and merge request author to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

**For code reviewer**

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

**For developper**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [x] Original issue have been updated with a short resume of implemented solution

**For quality team**

- [x] Tested by quality team

### CODE REVIEW

**Comment by PhilippeAccorsi** [comment]
For testing I need postgres database or I can see creation of index on sqlite database ?

**Comment by lebouquetin** [comment]
@PhilippeAccorsi I hope indexes are build on all database systems

**Comment by inkhey** [comment]
> @PhilippeAccorsi I hope indexes are build on all database systems. @inkhey could you confirm?

I confirm, the only specific database behaviour is when upgrading a mysql database, which automatically create index on foreign key, we do not recreate index that already exist.

File: backend/tracim_backend/models/data.py [hunk 1]
```diff
@@ -851,6 +851,8 @@ def has_new_information_for(self, user: User) -> bool:
 # TODO - G.M - 2018-06-177 - [author] Owner should be renamed "author"
 Index("idx__content_revisions__owner_id", ContentRevisionRO.owner_id)
 Index("idx__content_revisions__parent_id", ContentRevisionRO.parent_id)
+Index("idx__content_revisions__content_id", ContentRevisionRO.content_id)
+Index("idx__content_revisions__workspace_id", ContentRevisionRO.workspace_id)
```
Comment (lebouquetin) [inline backend/tracim_backend/models/data.py:855]: Please add a comment explaining that on mysql the index may have a different name

**Comment by PhilippeAccorsi** [comment]
Tested and verified on SQLite database. Downgrade working correctly (tested on sqlite database).
Fix tested with patch on production and working good (postgresql database).

File: backend/tracim_backend/models/data.py (hunk 1 above)
Comment (inkhey) [reply backend/tracim_backend/models/data.py:858]: done

**Pull request merged**